dcos-commons-tls-nginx/assets/http.conf

```diff
@@ -0,0 +1,13 @@
+server {
+    listen       80 default_server;
+    listen       [::]:80 default_server;
+    server_name  nginx-https.hello-world.l4lb.thisdcos.directory;
+    root         /usr/share/nginx/html;
+
+    if ($http_x_forwarded_proto = 'http') {
+    return 301 https://$server_name$request_uri;
+    }
+
+    location / {
+    }
+}
```

Comment:
If it is not required by any of the tests it shouldn't be there. Not listening on port 80 and not serving HTTP at all is a more direct way of preventing false positives :-)

But hey.. :-) :rocket: ;)